test/save_assemblies_from_fastas_integration_test.py

```diff
@@ -622,7 +622,8 @@ def test_parallelize_import_fasta_mass(config, impl, context, scratch):
         {'GC content': '0.41457', 'Size': '4096482', 'N Contigs': '1', 'MD5': '949a0fe665048cb917c8cf74f75c74b7'},
```

Comment:
Are there integration tests that don't include metadata?

Reply:
yea. This test previously imports 6 files. I updated one of them with `object_metadata`.  Other 5 inputs do not have metadata. And there are other tests like `test_assembly_objects_generator_overflow` that doesn't include metadata.

Reply:
:+1: